packages/paste-core/components/alert-dialog/src/AlertDialog.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {useUID} from '@twilio-paste/uid-library';
+import {useTransition} from '@twilio-paste/animation-library';
+import {css, styled} from '@twilio-paste/styling-library';
+import {safelySpreadBoxProps} from '@twilio-paste/box';
+import {ModalDialogOverlay, ModalDialogContent} from '@twilio-paste/modal';
+import {AlertDialogHeader} from './AlertDialogHeader';
+import {AlertDialogBody} from './AlertDialogBody';
+import {AlertDialogFooter} from './AlertDialogFooter';
+
+export const NarrowModalDialogContent = styled(ModalDialogContent)(() =>
```

Comment:
If we're now exporting this from the package, I don't love the name. Can it just be `AlertDialogContent`?